### PR TITLE
Sidebar

### PR DESCRIPTION
**Pull request by cyyan88**

### CODE REVIEW

File: src/components/layout/SideBar.jsx [hunk 1]
```diff
@@ -13,6 +13,12 @@ const { Sider } = Layout;
 
 const SideBar = props => {
 	const { context, dispatch } = useContext(Context);
+	
+	const [winWidth, setWinWidth] = useState(window.innerWidth);
+	function change(wid){ setWinWidth(wid) }
+	let headStyles = winWidth < 768 ? {fontSize: '1.15em'} : {fontSize: '2vw'}
```
Comment (nfried16) [inline src/components/layout/SideBar.jsx]: why not just use true/false for this winWidth state variable? It seems it is either a quarter of the screen or full screen, no in between?

File: src/components/layout/SideBar.jsx [hunk 2]
```diff
@@ -13,6 +13,12 @@ const { Sider } = Layout;
 
 const SideBar = props => {
 	const { context, dispatch } = useContext(Context);
+	
+	const [winWidth, setWinWidth] = useState(window.innerWidth);
+	function change(wid){ setWinWidth(wid) }
```
Comment (nfried16) [inline src/components/layout/SideBar.jsx]: This function is unnecessary, you can just use setWinWidth everywhere you use change

File: src/components/layout/AddModal.jsx [hunk 3]
```diff
@@ -46,8 +46,8 @@ const AddModal = props => {
 	return (
 		<span>
 			<Menu.Item key="4" className="menu-item" onClick={showModal}>
-				<span style={{ display: 'flex', alignItems: 'center' }}>
-					<FileAddFilled style = {{margin: '0 0.5rem 0 0'}}/>
+				<span style={{ display: 'flex', alignItems: 'center'}, props.style}>
```
Comment (nfried16) [inline src/components/layout/AddModal.jsx:49]: what is this added for?

Comment (cyyan88) [reply src/components/layout/AddModal.jsx:49]: For the "Add Dashboard" on the sidebar I couldn't find where to change the font size in the SideBar.jsx so I passed in the font sizes based on winWidth as a prop, idk if there'd be a better way to do that 😓

**Review by nfried16 — APPROVED** [review]

**Pull request merged**